lib/acm_bot.ex

```diff
@@ -8,7 +8,8 @@ defmodule AcmBot do
 
     children = [
       ExGram,
-      {AcmBot.Bot, [method: :polling, token: token]}
+      {AcmBot.Bot, [method: :polling, token: token]},
+      AcmBot.Cache
```

Comment:
I recommend to start the Cache before the Bot, in that case you won't have any problems like receiving an update with a file and trying to go to the Cache and that it doesn't exists